src/modules/roc_address/target_posix/roc_address/socket_addr_enums.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2019 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+//! @file roc_address/target_posix/roc_address/socket_addr_enums.h
+//! @brief Socket Address enums.
+
+#ifndef ROC_ADDRESS_SOCKET_ADDR_ENUMS_H_
+#define ROC_ADDRESS_SOCKET_ADDR_ENUMS_H_
+
+namespace roc {
+namespace address {
+
+//! Address type.
+enum AddrType { IP4, IP6 };
```

Comment:
Can we use `IPv4` and `IPv6` for consistency with the rest code? Also maybe better name it `AddrFamily`, it's a more correct/precise term I think.